src/renderer/wg_engine/tvgWgCompositor.cpp

```diff
@@ -268,16 +275,22 @@ void WgCompositor::composeShape(WgContext& context, WgRenderDataShape* renderDat
     beginRenderPass(commandEncoder, &storageInterm, true);
     drawShape(context, renderData, WgPipelineBlendType::Custom);
     endRenderPass();
-    // run compisition
-    compose(commandEncoder, &storageInterm, mask, target, composeMethod);
     // restore current render pass
     beginRenderPass(commandEncoder, target, false);
+    // apply mask
```

Comment:
I'm confused here. this function is used only for clipping, so why this comment? also the arg name 'mask'? should it be clip? composite? 
Could it just be a matter of nomenclature? is it consistent with the sw engine?

Reply:
There are a different nameing convention and it will be handled in the future and resolved